app-modules/service-management/database/migrations/2023_09_01_172133_create_service_request_form_submissions_table.php

```diff
@@ -48,6 +48,7 @@ public function up(): void
             $table->foreignUuid('service_request_priority_id')->nullable()->constrained('service_request_priorities');
             $table->string('author_id')->nullable();
             $table->string('author_type')->nullable();
+            $table->text('description')->nullable();
```

Comment:
This will need to be added in a separate migration. Including a `down` method and we will need to ensure that the app will not break if the code is deployed before the migration is run.